ui/job-view/details/PinBoard.jsx

```diff
@@ -414,6 +414,8 @@ class PinBoard extends React.Component {
                     {job.job_type_symbol}
                   </span>
                   <span
+                    role="button"
```

Comment:
For this one, I modified the `Button` to be like this:

                    className={`pinned-job-close-btn mr-1 mb-1 ${
                      selectedJobId === job.id ? 'selected-job' : ''
                    }`}
                    size={selectedJobId === job.id ? 'large' : 'small'}
                    onClick={() => unPinJob(job)}
                    color="secondary"
                    outline
                    title="un-pin this job"
                  >
                    <FontAwesomeIcon icon={faTimes} title="Unpin job" />
                  </Button>